libs/velo-external-db-core/src/utils/schema_utils.ts

```diff
@@ -0,0 +1,123 @@
+import { AdapterOperators } from '@wix-velo/velo-external-db-commons'
+import { InputField, ResponseField } from '@wix-velo/velo-external-db-types'
+import { Field, FieldType, QueryOperator } from '../spi-model/collection'
+const { eq, ne, string_contains, string_begins, string_ends, gt, gte, lt, lte, include } = AdapterOperators
+
```

Comment:
I'm not sure that file is needed, maybe all of them should be private methods under the schema service.
but either way, I think you should create unit tests for all of these functions.